### PR TITLE
feat(config): add support for HE-FT01, small fixes for HE-HT01

### PR DESCRIPTION
**Pull request by popescunsergiu**
feat(config): add support for he-ft01
fix(config): small fixes for he-ht01

### CODE REVIEW

File: packages/config/config/devices/0x0344/he-ht01.json [hunk 1]
```diff
@@ -119,157 +119,123 @@
...
-			"defaultValue": 210
+			"minValue": 0,
+			"maxValue": 1100,
+			"defaultValue": 0
 		},
 		{
 			"#": "13",
-			"label": "Air Temperature: Maximum",
-			"unit": "0.1 °C",
+			"label": "Fan Medium Speed Relay Load Power",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0344/he-ht01.json]: I don't understand this label.

File: packages/config/config/devices/0x0344/he-ht01.json [hunk 2]
```diff
@@ -119,157 +119,123 @@
...
-			"defaultValue": 270
+			"minValue": 0,
+			"maxValue": 1100,
+			"defaultValue": 0
 		},
 		{
 			"#": "14",
-			"label": "Floor Temperature: Minimum",
-			"unit": "0.1 °C",
+			"label": "Fan High Speed Relay Load Power",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0344/he-ht01.json]: dito

File: packages/config/config/devices/0x0344/he-ht01.json [hunk 3]
```diff
@@ -119,157 +119,123 @@
...
-			"defaultValue": 180
+			"minValue": 0,
+			"maxValue": 1100,
+			"defaultValue": 0
 		},
 		{
 			"#": "15",
-			"label": "Floor Temperature: Maximum",
-			"unit": "0.1 °C",
+			"label": "Heating Valve Relay Load Power",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0344/he-ht01.json]: dito

File: packages/config/config/devices/0x0344/he-ht01.json [hunk 4]
```diff
@@ -119,157 +119,123 @@
...
+			"defaultValue": 0
 		},
 		{
 			"#": "16",
-			"label": "Floor Temperature Calibration",
-			"unit": "0.1 °C",
-			"valueSize": 1,
-			"minValue": -100,
-			"maxValue": 100,
+			"label": "Cooling Valve Relay Load Power",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0344/he-ht01.json]: dito

**Review by AlCalzone — CHANGES_REQUESTED** [review]
I think you mixed up the filenames. Otherwise mostly looks good.

File: packages/config/config/devices/0x0344/he-ht01.json (hunk 1 above)
Comment (popescunsergiu) [reply packages/config/config/devices/0x0344/he-ht01.json]: This was my mistake. I've worked on FT01 and fixed some HT01 params but I've reversed the file names in the process

I also added description to the params

File: packages/config/config/devices/0x0344/he-ht01.json (hunk 2 above)
Comment (popescunsergiu) [reply packages/config/config/devices/0x0344/he-ht01.json]: Same as above

File: packages/config/config/devices/0x0344/he-ht01.json (hunk 3 above)
Comment (popescunsergiu) [reply packages/config/config/devices/0x0344/he-ht01.json]: Same as above

File: packages/config/config/devices/0x0344/he-ht01.json (hunk 4 above)
Comment (popescunsergiu) [reply packages/config/config/devices/0x0344/he-ht01.json]: Same as above

**Comment by popescunsergiu** [comment]
> I think you mixed up the filenames. Otherwise mostly looks good.

Fixed it, sorry for the issue

**Review by AlCalzone — APPROVED** [review]
I've reworked the description again. After reading the part in the manual twice, it made more sense :D

**Pull request merged**